casper/src/main/scala/coop/rchain/casper/Casper.scala

```diff
@@ -215,6 +213,11 @@ sealed abstract class MultiParentCasperInstances {
         _runtime.close()
       }
 
+      def normalizedInitialFault(weights: Map[Validator, Int]): F[Float] =
+        (equivocationsTracker.map(_.equivocator).toSet.map(weights).sum.toFloat / weightMapTotal(
```

Comment:
Perhaps this should be a `flatMap` over `weights.get` instead? Wouldn't want it to error out after the equivocator is rotated out of the weights map.

Reply:
Thanks for the catch